packages/maskbook/src/social-network-adaptor/twitter.com/utils/selector.ts

```diff
@@ -15,6 +15,42 @@ const querySelectorAll = <T extends E>(selector: string) => {
     return new LiveSelector().querySelectorAll<T>(selector)
 }
 
+//#region "Enhanced Profile"
+export const searchProfileSelector: () => LiveSelector<E, true> = () =>
+    querySelector<E>('[aria-label="Profile timelines"][role="navigation"]')
```

Comment:
You should not rely on `aria-label` because it is different in different natural languages, for example, it will not work for Chinese or Japanese users because they have `aria-label="プロファイルタイムライン"`.

This suggestion also applies to other new selectors in this PR.